.evergreen/setup-environment.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
```

Comment:
```suggestion
#! /usr/bin/env bash
```
just for [cross compat reasons](https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash)